app/src/main/java/net/gsantner/markor/util/AppSettings.java

```diff
@@ -29,34 +32,38 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
-import java.util.Comparator;
 import java.util.List;
 import java.util.Locale;
 import java.util.Random;
 
-@SuppressWarnings({"SameParameterValue", "WeakerAccess", "FieldCanBeLocal", "unused"})
+import other.de.stanetz.jpencconverter.PasswordStore;
+
+import static net.gsantner.markor.format.TextFormat.FORMAT_UNKNOWN;
+
+@SuppressWarnings({"SameParameterValue", "WeakerAccess", "FieldCanBeLocal"})
 public class AppSettings extends SharedPreferencesPropertyBackend {
     private final SharedPreferences _prefCache;
     private final SharedPreferences _prefHistory;
     public static Boolean _isDeviceGoodHardware = null;
+    private final ContextUtils _contextUtils;
```

Comment:
Created a class level `_contextUtils` since it is used in multiple places.

@gsantner I have not called `freeContextRef`. All it seems to do is set the internal context reference to null. Is it really necessary?

Reply:
?

Reply:
not strictly, but generally speaking no references to Context's and Activity's  is better than references that cannot be freed.